applications/StructuralMechanicsApplication/python_scripts/adaptative_remeshing_structural_mechanics_analysis.py

```diff
@@ -2,15 +2,19 @@
 
 # Importing Kratos
 import KratosMultiphysics as KM
-import KratosMultiphysics.StructuralMechanicsApplication as SMA
 
 # Other imports
 import sys
-from KratosMultiphysics.StructuralMechanicsApplication import python_solvers_wrapper_adaptative_remeshing_structural
 
 # Import the base structural analysis
 from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis as BaseClass
 
+# Import solver wrapper
+from KratosMultiphysics.StructuralMechanicsApplication import python_solvers_wrapper_adaptative_remeshing_structural
```

Comment:
did structural mechanics approve?

Reply:
@KratosMultiphysics/implementation-committee did it three months ago

Reply:
I think this is ok since it is the remeshing-analysis